frontend/packages/dev-console/src/components/helm/HelmReleaseNotes.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { match as RMatch } from 'react-router';
+import { LoadingBox } from '@console/internal/components/utils';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { SyncMarkdownView } from '@console/internal/components/markdown-view';
+import { HelmRelease } from './helm-types';
+
+export interface HelmReleaseNotesProps {
+  match: RMatch<{
+    ns?: string;
+    name?: string;
+  }>;
+}
+
+type LoadedHelmReleaseNotesProps = {
+  helmReleaseNotes: string;
+};
+
+export const LoadedHelmReleaseNotes: React.FC<LoadedHelmReleaseNotesProps> = ({
+  helmReleaseNotes,
+}) => (
+  <div className="co-m-pane__body">
+    <SyncMarkdownView content={helmReleaseNotes} />
+  </div>
+);
+
+const HelmReleaseNotes: React.FC<HelmReleaseNotesProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  const helmReleaseName = match.params.name;
+  const [helmReleaseNotes, setHelmReleaseNotes] = React.useState<string>(null);
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const fetchHelmReleases = async () => {
+      let res: HelmRelease[];
+      try {
+        res = await coFetchJSON(`/api/helm/releases?ns=${namespace}`);
```

Comment:
Each tab in the Helm release details page seems to be fetching the same release details. And it does that every time we change tabs. I know this is an issue with the `DetailsPage` component itself where we cannot pass any props to the tab components. But I was thinking is there a way we could re-use the same data and avoid fetching this again? Definitely not a blocker for this PR but we should probably log a ticket or something to look at this later.

Reply:
I agree.
These pages were meant for k8s resources and there are a couple ways data is passed along:
- `obj.data`
- `customData`

It would be nice to be able fetch this info once in `HelmReleaseDetailsPage` and pass the data along to the tabs.

Reply:
Done.